scripts/check-cabal.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Note that the workflow currently tries to execute `scripts/check-cabal-files.sh` instead of this.

Also, the other scripts are in the `ci` subdirectory of `./scripts` instead of directly in it.

Reply:
Fixed!